tools/lib/h5diff.c

```diff
@@ -992,24 +969,6 @@ h5diff(const char *fname1, const char *fname2, const char *objname1, const char
     }
     H5TOOLS_DEBUG("groups traversed - errstat:%d", opts->err_stat);
 
-#ifdef H5_HAVE_PARALLEL
-    if (g_Parallel) {
-        int i;
-
-        if ((strlen(fname1) > MAX_FILENAME) || (strlen(fname2) > MAX_FILENAME)) {
-            fprintf(stderr, "The parallel diff only supports path names up to %d characters\n", MAX_FILENAME);
-            MPI_Abort(MPI_COMM_WORLD, 0);
-        } /* end if */
-
-        strcpy(filenames[0], fname1);
-        strcpy(filenames[1], fname2);
-
-        /* Alert the worker tasks that there's going to be work. */
-        for (i = 1; i < g_nTasks; i++)
-            MPI_Send(filenames, (MAX_FILENAME * 2), MPI_CHAR, i, MPI_TAG_PARALLEL, MPI_COMM_WORLD);
-    } /* end if */
```

Comment:
Moved this block down to get most/all of the serial output out of the way before beginning parallel operations.